src/dtls_stream.rs

```diff
@@ -9,81 +9,33 @@ use std::{fmt, io};
 /// and both the server and the client are ready for receiving and sending
 /// data. Bytes read from a `DtlsStream` are decrypted from `S` and bytes written
 /// to a `DtlsStream` are encrypted when passing through to `S`.
-pub struct DtlsStream<S>(ssl::SslStream<S>);
+pub struct DtlsStream<C>(pub(crate) C);
```

Comment:
Do we need to make this public since we have the SyncStream now

Reply:
as far as I undertsand - yes